gradle.properties

```diff
@@ -0,0 +1,8 @@
+version=2.1.1-SNAPSHOT
+# Knot.x Docker image version (https://hub.docker.com/r/knotx/knotx)
+knotx.version=2.1.0
+knotx.conf=knotx
+docker.image.name=knotx/knotx-docker
```

Comment:
What for is that variable?

Reply:
That variable is redundant. I deleted it.